__tests__/sagas-test.ts

```diff
@@ -0,0 +1,18 @@
+import { runSaga } from 'redux-saga';
+import { onGetCounters } from '../src/sagas';
+import { getCounters, getCountersSuccess } from '../src/actions';
+import fetchMock from 'fetch-mock';
+
+it('will fetch counters - success', async () => {
+  const expectedCounters = [{ count: 6 }, { count: 4 }, { count: 2 }, { count: 1 }];
+  fetchMock.mock('https://api.myjson.com/bins/ywmke', expectedCounters);
```

Comment:
Good job mocking the API. Just curious, did you ever get the way I mock APIs working? (I don't expect you to change your mock in this learning project, I'm just curious.)

Reply:
Wasn't able to get it to work, unfortunately 😢

Reply:
Lame. :( But understandable, I feel like setting up a project is a dark art that is only performed once and hopefully never again.